app/javascript/dashboard/routes/dashboard/settings/inbox/channels/instagram/DuplicateInboxBanner.vue

```diff
@@ -10,7 +10,10 @@ defineProps({
 </script>
 
 <template>
-  <Banner color="ruby">
-    {{ content }}
+  <Banner color="amber">
+    <div class="flex items-center gap-2">
+      <div class="flex"><fluent-icon icon="info" size="16" /></div>
```

Comment:
@muhsin-k Let us use Lucide icons instead of old `<fluent-icon />`
```suggestion
       import Icon from 'dashboard/components-next/icon/Icon.vue';
       <Icon icon="i-lucide-info" class="size-4" />
```
**OR**
```suggestion
       <span class="i-lucide-info size-4" />
```